modules/spring-config/src/main/resources/META-INF/mule-module.xsd

```diff
@@ -150,24 +150,42 @@
                         </xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
-                <xsd:attribute name="namespace" use="required">
+                <xsd:attribute name="category" type="categoryType" default="COMMUNITY">
                     <xsd:annotation>
                         <xsd:documentation>
-                            Expected namespace of the module to look for when generating the schemas.
+                            Expected category of the module.
                         </xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
-                <xsd:attribute name="category" type="categoryType" default="COMMUNITY">
+                <xsd:attribute name="vendor" type="xsd:string" default="MuleSoft">
                     <xsd:annotation>
                         <xsd:documentation>
-                            Expected category of the module.
+                            Expected vendor of the module.
```

Comment:
Can you explain a little bit more what vendor is?. If I a new developer and just do an extension for me or my company, what should I put here?